api/telemetry/v1alpha1/otlp_config.go

```diff
@@ -40,7 +42,19 @@ type QueueSettings struct {
 
 	// StorageID if not empty, enables the persistent storage and uses the component specified
 	// as a storage extension for the persistent queue
-	StorageID string `json:"storage,omitempty"` //TODO this is *component.ID at Otel
+	// WARNING: This field will be always overwritten by the operator.
+	Storage *string `json:"storage,omitempty"`
+}
+
+func (q *QueueSettings) SetDefaultQueueSettings() {
+	if q == nil {
+		q = &QueueSettings{}
+	}
+	q.Enabled = true
+	q.Storage = utils.ToPtr(storage.DefaultFileStorageName.String())
+	if q.QueueSize == 0 || q.QueueSize < 0 {
+		q.QueueSize = 1000
```

Comment:
[Nit]
I would add a comment to the `QueueSize` field mentioning the default value to be 1000.